Services/NewsService/Startup.cs

```diff
@@ -34,7 +40,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 endpoints.MapControllers();
             });
 
-
+            app.UseSwagger();
+            app.UseSwaggerUI(c =>
+            {
+                c.SwaggerEndpoint("/swagger/v1/swagger.json", "NewsFuckingPublisher");
```

Comment:
I'm not sure tah all these are supposed to be merged into develop

Reply:
deleted